examples/zstd.rs

```diff
@@ -5,31 +5,36 @@ fn main() {
     use oci_spec::image::Platform;
     use ocidir::{cap_std::fs::Dir, new_empty_manifest, OciDir};
     let dir = Dir::open_ambient_dir(env::temp_dir(), ocidir::cap_std::ambient_authority()).unwrap();
```

Comment:
```suggestion
    let dir = Dir::open_ambient_dir(env::temp_dir(), ocidir::cap_std::ambient_authority()).unwrap();
    let tmpdir = "zstd-example";
    dir.create_dir_all(tmpdir).unwrap();
    let dir = dir.open_dir(tmpdir).unwrap();
```
I suggest creating a separate directory in `/tmp` for the oci directory, otherwise we end up with `/tmp/index.json` and `/tmp/blobs/sha256`.